tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "include": ["lib/**/*"],
+  "exclude": ["node_modules", "test/**/*", "webpack.*.js"],
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "allowJs": true,
+    "declaration": true,
+    "declarationDir": "dist/@types",
+    "emitDeclarationOnly": true,
+    "newLine": "lf",
+    "outDir": "./dist",
+    "isolatedModules": true,
+    "strict": true,
+    "noImplicitAny": true,
+    "moduleResolution": "node",
+    "types": ["node"],
```

Comment:
This library does not run in node but in browsers so this may lead to tricky bugs